chore(curriculum): add colored markers to new cert (#56718)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/61764c602bee6974e7790f35.md

---
id: 61764c602bee6974e7790f35
title: Step 11
challengeType: 0
dashedName: step-11
---

# --description--

> To give the markers different colors, you will need to add a unique class to each one. Multiple classes can be added to an element by listing them in the `class` attribute and separating them with a space. For example, the following adds both the `animal` and `dog` classes to a `div` element.

Reviewer: ```suggestion
To give the markers different colors, you’ll need to add a unique class to each one. You can add multiple classes to an element by listing them in the `class` attribute and separating them with a space. For example, the following code adds both the `animal` and `dog` classes to a `div` element:
```
